client/login/ResetPassword/ResetPassword.js

```diff
@@ -0,0 +1,106 @@
+import React, { useState, useCallback } from 'react';
+import { Button, TextInput, Field, Modal, Box, Throbber } from '@rocket.chat/fuselage';
+import { useSafely } from '@rocket.chat/fuselage-hooks';
+
+import { useTranslation } from '../../contexts/TranslationContext';
+import { useUser } from '../../contexts/UserContext';
+import { useMethodData, useMethod } from '../../contexts/ServerContext';
+import { useRouteParameter } from '../../contexts/RouterContext';
+
+const getChangePasswordReason = ({
+	requirePasswordChange,
+	requirePasswordChangeReason = requirePasswordChange
+		? 'You_need_to_change_your_password'
+		: 'Please_enter_your_new_password_below',
+} = {}) => requirePasswordChangeReason;
+
+
+const ResetPassword = ({ resetPassword }) => {
+	const user = useUser();
+	const t = useTranslation();
+	const [{ enabled: policyEnabled, policy: policies } = {}] = useMethodData('getPasswordPolicy');
```

Comment:
This method tries to get `userId`, but since is a password recovery, it doesn't exist yet, and I can't change the password.
![image](https://user-images.githubusercontent.com/40830821/87189092-6c080700-c2c6-11ea-94a1-13e187beebde.png)
![image](https://user-images.githubusercontent.com/40830821/87189116-76c29c00-c2c6-11ea-8d6a-56c9195ed3f2.png)